### PR TITLE
added 5 .html and 5 .js for week2 javascript homework.

### PR DESCRIPTION
**Pull request by TITHI1244**

### CODE REVIEW

File: javascript/javascript1/week2/event application.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            returnedDay = "Saturday";
+            break;
+    }
+    return returnedDay;
+}
+function getRemainder(days) {
+    return days % 7;
+}
+
+let currentDayIndex = new Date().getDay();
```
Comment (aina21) [inline javascript/javascript1/week2/event application.js:33]: try wth const

File: javascript/javascript1/week2/event application.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const remainingDays = 9;
+let eventDayIndex = getRemainder(remainingDays + currentDayIndex);
```
Comment (aina21) [inline javascript/javascript1/week2/event application.js:35]: const

File: javascript/javascript1/week2/student manager.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// Student manager
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if(studentName !== "") {
+        if(class07Students.length === 0) {
+            class07Students.push(studentName);
+        } else if(class07Students.length < 6) {
+            for(let i = 0; i <= class07Students.length; i++) {
```
Comment (aina21) [inline javascript/javascript1/week2/student manager.js:8]: nice, you can also use include(). read more about it here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week2/student manager.js (hunk 3 above)
Comment (TITHI1244) [reply javascript/javascript1/week2/student manager.js:8]: Got it. Thanks :)

File: javascript/javascript1/week2/event application.js (hunk 1 above)
Comment (TITHI1244) [reply javascript/javascript1/week2/event application.js:33]: Yes, my bad :)

File: javascript/javascript1/week2/event application.js (hunk 2 above)
Comment (TITHI1244) [reply javascript/javascript1/week2/event application.js:35]: Sure :)

**Pull request merged**